src/logic/safe/transactions/gas.ts

```diff
@@ -92,6 +93,141 @@ export const estimateTxGasCosts = async (
   }
 }
 
+const getOpenEthereumErrorDataResult = (errorMessage: string): string | undefined => {
+  // Extracts JSON object from the error message
+  const [, ...error] = errorMessage.split('\n')
+  const errorAsJSON = JSON.parse(error.join(''))
+
+  if (errorAsJSON?.data) {
+    const [, dataResult] = errorAsJSON.data.split('Reverted ')
```

Comment:
Maybe instead if split by `Reverted` we split by ` ` (space), this way we also handle Nethermind (which uses `revert <hex data>`

Reply:
Do you have a nethermind endpoint rpc that I can use for testing the response?

Reply:
Hmmm seems they switched back .... annoying ... so we can only test this in our tests with the data from the issue ... should be enough for now :/

Reply:
I refactored the file to support nethermind and added a test case for it :)